application/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "scripts": {
+        "login": "npm login --scope=@codbex --auth-type=legacy --registry=https://npm.pkg.github.com"
+    },
+    "dependencies": {
+        "@codbex/codbex-accounts": "0.1.0",
```

Comment:
Are these all of the needed dependencies? Don't we have to add the `codbex-athena` dependency and also the dependencies from the `project.json` file from here: https://github.com/codbex/codbex-athena-dashboard/blob/main/codbex-athena/project.json#L3-L28?